apps/summerfi-api/lib/sdk-router-function/e2e/refinance-client.spec.ts

```diff
@@ -88,9 +88,9 @@ describe('Refinance Client-Server Communication', () => {
     }
 
     const refinanceParameters: RefinanceParameters = {
-      sourcePosition: sourcePosition,
+      position: sourcePosition,
       targetPool: pool,
-      slippage: Percentage.createFrom({ percentage: 20.5 }),
+      slippage: 0.1,
```

Comment:
Why are we using a number instead of Percentage here?

Reply:
Yeah this is something @paszkowskiDamian implemented in simulator service so I would need to change that service which is not relevant to this PR and should be handled separately maybe